variables.tf

```diff
@@ -164,6 +164,17 @@ variable "send_job_alerts_daily_email_task_schedule" {
   default     = "cron(0 08 * * ? *)"
 }
 
+variable "send_feedback_prompt_email_task_command" {
+  description = "The Entrypoint for the send_feedback_prompt_email task"
+  default     = ["rake", "verbose", "feedback_prompt_email:send"]
+}
+
+variable "send_feedback_prompt_email_task_schedule" {
+  description = "send_feedback_prompt_email schedule expression - https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html"
+  default     = "cron(0 09 * * ? *)"
```

Comment:
Set this to run at 9 every day but this was just my best guess at a suitable time. Could this be a different time and ideally when should this cron be run?

Reply:
Could be at any time 🙂.

9am sounds sensible to me, but maybe ask in the #tv_product room for any suggestions otherwise.

Also be aware, what does this config do in regard to summer time? Is it 9 UTC or 9 local time?

Reply:
@adrianclay The cron times are in UTC as per https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html